docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -0,0 +1,75 @@
+[id='pro-running-the-published-repositories-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-cloning-separate-repositories-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-repository-npm-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-mongodb-and-redis-{chapter}[Starting MongoDB and Redis]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+
+= Prerequisites
+
+* Node and npm installed (tested on Node {WFM-RC-NodeVersion})
+* git
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
+* MongoDB (tested with version: {WFM-RC-MongoDBVersion})
+* Redis (tested with version: {WFM-RC-RedisVersion})
+
+= Running the Demo App from the Published Repositories
+
+This procedure will install the latest published version of {WFM-RC-NameShort} from the separate, per-demo-app GitHub repositories.
+
+[id='pro-cloning-separate-repositories-{chapter}']
+[discrete]
+== Cloning the Git Repositories
+
+[source,bash]
+----
+git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
```

Comment:
We need to call here explicity what branch we want to use - master for recent changes. 
We may have releases that are RC1 so people should be never pointed to master.
Just current released version that will be a tag.

Reply:
Done!

Reply:
via 43410db3d611c8d43ff6160a825f79429e2c7fc0